### ISSUE TITLE
Get cookies from browser

### ISSUE THREAD

**Issue by ltworf** [ltworf/localslackirc#363]
ty-dlp does this thing of getting the cookies it needs from the browser, so the user isn't hassled to do strange procedures to extract them.

It would be cool to support the same, like a tool to create a configuration file.

**Comment by MikeRich88** [comment]
Wouldn't you have to support each browser separately then?

Safari on Mac probably uses sqlite database to store the cookies.

**Comment by ltworf** [comment]
Yes, but by "each browser separately" I mean "just firefox because that's what I use and other people can contribute" :)
